src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java

```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.elements.mercury.components;
 
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.contentpatterns.MercurySubpages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)